staging_service/app.py

```diff
@@ -19,24 +19,22 @@
 
 
 @routes.get("/importer_mappings/{query:.*}")
-async def importer_mappings(request: web.Request) -> web.json_response:
-    raise web.HTTPBadRequest(text="GET not supported. Use POST")
-
-
-@routes.post("/importer_mappings/{query:.*}")
 async def importer_mappings(request: web.Request) -> web.json_response:
     """
     Return a dictionary with two lists: apps and mappings
     Apps are a list of importers
     Mappings are a list of mapping between passed in files, and available apps
     :param request: contains a list of files e.g. ['file1.txt','file2.fasta']
     """
-    file_list = parse_qs(await request.text()).get("file_list", [])
+
+    file_list = parse_qs(request.query_string).get("file_list", [])
     if len(file_list) == 0:
-        raise web.HTTPBadRequest(text="must provide file_list field ")
```

Comment:
I don't think this case was tested

Reply:
Oh I see, the test was made less stringent. Why not test that the query string is included in the result though?

Reply:
I don't understand. There is no query string when it reaches this HTTPBadRequest, so how can we test that it is included in the error?

Reply:
The error message is now

```
f"must provide file_list field. Your provided qs: {request.query_string}"
```

But the tests only check the first sentence, and the query string is known: https://github.com/kbase/staging_service/pull/95/files/035afdf822a357383909cabfbb1eb0c65c9b987d#diff-a67cb1853203a6f1956991a9d9881d231c4d43557f5baffd45cc672a87e41cc6R962

So it seems to me the whole string could be tested, which would ensure the error is being reported correctly

Reply:
Ok done